contracts/eosio.bios/include/eosio.bios/eosio.bios.hpp

```diff
@@ -78,8 +81,13 @@ namespace eosiobios
     */
    class [[eosio::contract("eosio.bios")]] bios : public eosio::contract
    {
+
    public:
       using contract::contract;
+      static constexpr eosio::symbol system_resource_currency = eosio::symbol("ONO", 4);
+      static constexpr eosio::name idtmy_name = "id.tmy"_n;
+      static constexpr eosio::name gov_name = "gov.tmy"_n;
+
```

Comment:
error: call to non-static member function without an object argument
      static constexpr eosio::name self_name = get_self();
                                               ^~~~~~~~

Reply:
ok